week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,55 @@
 /** Write your CSS in here **/
+body{
+    background: rgb(245, 171, 33);
+  }
+  h3{
+      color:  rgb(245, 171, 33);
+      font-size: 24px;
+      text-align: end;
+  }
+  p{
+  
+    margin:100px 90px;
+    padding:30px 30px;
+    color:rgb(245, 171, 33);
+    align-self: flex-end;
+    margin:0px 20px 10px 0px;
+    
+  
+  }
+  #container{
+    background: white;
+    margin:100px 90px;
+    padding:40px 50px;
+    display:flex;
+    flex-direction: column;
+  
+  
+  }
+
+  #quoteBox{
+      background-color: white;
+      width: 95%;
+      position: center;
+      margin:30px
+  }
+  #content{
```

Comment:
In general, try and avoid using IDs in your CSS. You can use IDs in your HTML and in your JS, but for CSS it's best practice to use classes.